integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestCreateTableWithSortScope.scala

```diff
@@ -68,10 +68,10 @@ class TestCreateTableWithSortScope extends QueryTest with BeforeAndAfterAll {
          | stringField STRING
          | )
          | STORED BY 'carbondata'
-         | TBLPROPERTIES('SORT_COLUMNS'='stringField', 'SORT_SCOPE'='BATCH_SORT')
+         | TBLPROPERTIES('SORT_COLUMNS'='stringField', 'SORT_SCOPE'='local_sort')
```

Comment:
already this suite has test case local sort, can remove this batch sort test case

Reply:
please remove test case about table 'tableWithBatchSort':
line 32: `sql("DROP TABLE IF EXISTS tableWithBatchSort")`
and the code from line 64.